source/slang/slang-dxc-support.cpp

```diff
@@ -82,7 +82,10 @@ namespace Slang
         // Now let's go ahead and generate HLSL for the entry
         // point, since we'll need that to feed into dxc.
         SourceResult source;
-        SLANG_RETURN_ON_FAIL(emitEntryPointSource(compileRequest, entryPointIndex, targetReq, CodeGenTarget::HLSL, endToEndReq, source));
+        List<Int> entryPointIndices;
+        entryPointIndices.add(entryPointIndex);
+        SLANG_RETURN_ON_FAIL(emitEntryPointsSource(compileRequest, entryPointIndices,
+            targetReq, CodeGenTarget::HLSL, endToEndReq, source));
```

Comment:
I would definitely advocate for introducing `emitEntryPointSource` (singular) back into the codebase so that call sites like this can be left unchanged, rather than having to build the singleton `List<Int>` in each case.